helpdesk_mgmt_ticket_tree_recent/__manifest__.py

```diff
@@ -3,17 +3,18 @@
 {
     "name": "Helpdesk Ticket Tree View order by recent",
     "summary": """
-        Sorts tickets on tree view by last update date.
+        Adds group that sorts tickets on tree view by last update date.
     """,
-    "version": "12.0.1.0.0",
+    "version": "12.0.1.0.1",
```

Comment:
Minor version is used for bug fixes, I think this is better option (this is a major improvement):
```suggestion
    "version": "12.0.2.0.0",
```